docs/ops.html

```diff
@@ -223,6 +223,15 @@ <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing C
     </li>
   </ul>
 
+  To reset offsets of a consumer group to latest offset, first make sure the instances are inactive, then:
+
+  <pre class="brush: bash;">
+  &gt; bin/kafka-consumer-groups.sh --bootstrap-server localhost:9092 --reset-offsets --group consumergroup1 --topic topic1 --to-latest
+
+  TOPIC                          PARTITION  NEW-OFFSET
+  topic1                         0          0
+  </pre>
+
```

Comment:
@tankhiwale  Thanks for the PR. I think we should list various sub-options  (--to-datetime, --by-duration, --to-earliest etc..) mentioned in KIP-122.  may be we can add one line explanation to each of the option.

Reply:
Acknowledged. Will update the PR

Reply:
Done. @omkreddy please review.

Reply:
Changes looks good to me.   looks like the PR contains additional commits.  It is good practice to create a new brach and commit changes to new branch and raise the PR.

https://cwiki.apache.org/confluence/display/KAFKA/Contributing+Code+Changes#ContributingCodeChanges-PullRequest

Reply:
@omkreddy I merged tankhiwale/trunk with apache/kafka/trunk. And hence i think those extra commits.
does something else need to be done from my end for this PR ? I see that the PR lists 2 files which I changed & no further conflicts as of now.

Reply:
LGTM 

cc @hachikuji

Reply:
@tankhiwale Can you address the below comment from hachikuji?